docs/install-and-setup.md

```diff
@@ -113,6 +107,26 @@ or:
 cargo install --locked --bin jj jj-cli
 ```
 
+
+#### Homebrew
```

Comment:
I honestly wonder if we should consolidate the Homebrew sections for Linux/Mac. Not a big deal I guess.

Reply:
If only homebrew supported Windows, I could put it and binstall in the same section...